lib/init/add-contributors-list.test.js

```diff
@@ -1,27 +1,25 @@
 import test from 'ava';
-import {addContributorsList} from './init-content';
+import {addContributorsList, setContentLanguage} from './init-content';
 
 test('should insert list under contributors section', t => {
   const content = [
     '# project',
     '',
     'Description',
     '',
-    '## Contributors',
+    '## Contributors<div id=\'contributors\'></div>',
```

Comment:
What's the purpose of these added `<div>`s? Is it so the anchor is consistent regardless of the language? I'm not sure that's necessary. Maybe I'm misunderstanding though :)